src/server/src/batch.cpp

```diff
@@ -126,6 +126,7 @@ process_batch(stinger_t * S, StingerBatch & batch)
   StingerServerState & server_state = StingerServerState::get_server_state();
   min_batch_ts = std::numeric_limits<int64_t>::max();
   max_batch_ts = std::numeric_limits<int64_t>::min();
+  int64_t chunksize = 8192;
```

Comment:
It would be nice if a) this number were somewhat generic most of the time and b) that the number was something like 64 or 128.